hrms/payroll/doctype/payroll_entry/payroll_entry.json

```diff
@@ -324,12 +325,21 @@
    "fieldtype": "Tab Break",
    "label": "Connections",
    "show_dashboard": 1
-  }
+  },
+  {
+   "default": "0",
+   "fieldname": "journal_entry_submitted",
+   "fieldtype": "Check",
+   "hidden": 1,
+   "label": "Journal Entry Submitted",
+   "read_only": 1,
+   "no_copy": 1
```

Comment:
On second thought, do we need this field at all? We are anyway checking if there are any existing JV accounts on load. We have `salary_slips_created` and `salary_slips_submitted` flags because these operations are queued for slips > 30. But in this case, JVs are created right away.